packages/framer-motion/src/value/types/utils.ts

```diff
@@ -0,0 +1,13 @@
+// If this number is a decimal, make it just five decimal places
+// to avoid exponents
+export const sanitize = (v: number) => Math.round(v * 100000) / 100000
+
+export const floatRegex = /(-)?([\d]*\.?[\d])+/g
+export const colorRegex =
+    /(#[0-9a-f]{6}|#[0-9a-f]{3}|#(?:[0-9a-f]{2}){2,4}|(rgb|hsl)a?\((-?[\d\.]+%?[,\s]+){2}(-?[\d\.]+%?)\s*[\,\/]?\s*[\d\.]*%?\))/gi
+export const singleColorRegex =
+    /^(#[0-9a-f]{3}|#(?:[0-9a-f]{2}){2,4}|(rgb|hsl)a?\((-?[\d\.]+%?[,\s]+){2}(-?[\d\.]+%?)\s*[\,\/]?\s*[\d\.]*%?\))$/i
+
+export function isString(v: any): v is string {
+    return typeof v === "string"
+}
```

Comment:
These are not really "type" utils, right? Can we come up with a better name?

Reply:
This folder is basically a drag-drop of `style-value-types`. The idea in the medium term is to give motion values "typed" data models - but not TypeScript types, more like https://developer.mozilla.org/en-US/docs/Web/API/CSS_Typed_OM_API. vs keeping values in strings as we currently do.

Maybe "model" or something would be a better name?

Reply:
Although it is pretty generic, I kind of like `model` 
I would also be ok with a comment explaining this on top of this file.